packages/esm-patient-orders-app/src/lab-results/lab-results-form-field.component.tsx

```diff
@@ -63,7 +84,7 @@ const ResultFormField: React.FC<ResultFormFieldProps> = ({ concept, control, def
               hideSteppers
               id={concept.uuid}
               key={concept.uuid}
-              label={concept?.display + printValueRange(concept)}
+              label={concept?.display + formatLabRange(concept)}
```

Comment:
Not an issue created by this PR, but I don't think this works the way we'd want if `concept` is actually null or undefined.

```suggestion
              label={`${concept?.display ? concept.display + ' ' : ''}${formatLabRange(concept)}`}
```